src/util/nya.rs

```diff
@@ -0,0 +1,51 @@
+pub mod nya {
+  use serde::Serialize;
+  use serde::Deserialize;
+  use std::fs::File;
+  use std::path::PathBuf;
+  use std::io::Read;
+
+
+  #[derive(Serialize, Deserialize, Debug)]
+  pub struct Stats {
+    #[serde(rename = "rl")]
```

Comment:
This is how we map an external JSON schema to our commonly used variables for Stats